browser/net/brave_network_audit_allowed_lists.h

```diff
@@ -69,8 +69,13 @@ inline constexpr const char* kAllowedUrlPrefixes[] = {
     "https://p3a.brave.com/",
     "https://star-randsrv.bsg.brave.com/",
 
+    // ads
+    "https://geo.ads.bravesoftware.com",
+    "https://static.ads.bravesoftware.com",
```

Comment:
why are these requests made before the user has opted into ads?

Reply:
cc: @tmancey @aseren

Reply:
@diracdeltas The ads component is used for search result ads regardless of whether a user has joined Brave Rewards, meaning it is always active. This change was enabled by default in brave-core 1.68.31, whereas it was previously enabled via Griffin. Thanks.

Reply:
I'm not sure I understand 

1. why this is happening in this test since IIRC it doesn't involve navigating to Brave Search, just opening the browser.
2. why this is triggering bravesoftware.com instead of brave.com, which is generally what we should be using in prod

Reply:
@diracdeltas the change that caused this network audit test to fail has been reverted. The network audit tests should now pass. cc @simonhong

Reply:
Ok, deleted.